units/provisioning_setup_host.go

```diff
@@ -795,6 +795,20 @@ func mountLinuxVolume(ctx context.Context, env evergreen.Environment, h *host.Ho
 		}))
 	}()
 
+	// check if we've done this already
+	cmd := client.CreateCommand(ctx).Append(fmt.Sprintf("ls %s/%s", h.Distro.HomeDir(), evergreen.HomeVolumeDir)).Background(true)
```

Comment:
given that this is a symlink can we just look at the output of mount for the name of the directory?

Reply:
done.
I thought it makes sense to check each command that can only run once (so if we've already mounted it but failed to make the symlink we'll get another chance).

Reply:
it occurred to me the directory won't be mounted at this point because we umount it earlier.
The problem must have been creating a duplicate symlink. I left in that check and removed the check for if the directory is already mounted.